spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+class ApplicationController < ActionController::Base
+  rescue_from CanCan::AccessDenied do |exception|
+    redirect_to user_groups_path, alert: exception.message
+  end
+end
+
+describe ApplicationController, type: :controller do
+  controller do
+    def index
+      raise CanCan::AccessDenied
+    end
+  end
+
+  describe "handle access denied" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.